dts/bindings/flash_controller/st,stm32-ospi-nor.yaml

```diff
@@ -67,3 +67,27 @@ properties:
       enum:
         - 1
         - 2
+    writeoc:
+      type: string
+      required: false
+      enum:
+        - "PP"            # Page program, PP (0x02) up to 256 bytes
```

Comment:
if you look into `dts/bindings/mtd/nordic,qspi-nor.yaml` file you can see different notation for the same kind of property.

Reply:
I agree on that you can use different solution in this PR. I just want to highlight other notation.

I would say that option 1. is outstanding IMO.
And yes this PR shouldn't be extended for reworking the others, so that topic for other patch as you mentioned.

Reply:
Considering the recent discovery... It seems that the `mode` doesn't really define the instruction and flash can use any instruction it likes for any mode. It is just a common practice to use certain opcodes for certain modes.

Therefore, instead of the read and write mode fields it shall be read and write opcodes with an extra dummy cycles field for the read.